contracts/permissionless/OrderbookReserve.sol

```diff
@@ -516,20 +509,20 @@ contract OrderbookReserve is OrderIdManager, Utils2, KyberReserveInterface, Orde
         return ethToTokenList.getOrderDetails(orderId);
     }
 
-    function calcKncStake(int weiAmount) public view returns(int) {
-        return(weiAmount * int(kncStakePerEtherBps) / 1000);
+    function calcKncStake(uint weiAmount) public view returns(uint) {
```

Comment:
looks ok, if possible please order the functions calcKncStake, calcBurnAmount, makerStakedKNC in a better order (according to who calls who). for example:
makerStakedKNC
makerUnlockedKnc
calcKncStake
calcBurnAmount

not must but easier to read..